__tests__/app.test.js

```diff
@@ -366,6 +366,64 @@ describe('PATCH /api/reviews/:review_id', () => {
     });
 });
 
+describe('QUERY GET /api/reviews', () => {
+  it(" default setings: category=allReviews, sort_by=date, order=decending", () => {
+    //do a request check if all reviews returned, sorted by date(use jest sort) in decending order
+    return request(app).get('/api/reviews').expect(200)
+    .then( result => result.body.reviews)
+    .then( reviews => {
+      expect(reviews).toHaveLength(13)
+      expect(reviews).toBeSorted( 'created_at', {decending: true})
+    })
+  })
+  it('returns reviews filtered by provided category', () => {
+    return request(app).get('/api/reviews?category=dexterity').expect(200)
+    .then( result => result.body.reviews)
+    .then( reviews => {
+      expect(reviews).toHaveLength(1)
+      expect(reviews).toBeSorted('created_at', {decending: true })
+     });
+  });
+
+  it('returns reviews sorted by provided sort_by column ', () => {
+    return request(app).get('/api/reviews?sort_by=title').expect(200)
+    .then( result => result.body.reviews)
+    .then( reviews => {
+      expect(reviews).toBeSorted('title', {decending: true })
+     });
+  });
+  it('returns reviews ordered by provideded order', () => {
+    return request(app).get('/api/reviews?order=ASC').expect(200)
+    .then( result => result.body.reviews)
+    .then( reviews => {
+      expect(reviews).toBeSorted('created_at', {decending: false })
+     });
+  });
```

Comment:
Please add another test with status 200 where the category is valid, but there are no reviews. For example: 'children's game'. This should respond with an empty array of reviews.